src/VestingWalletWithCliff.sol

```diff
@@ -14,20 +14,13 @@ abstract contract VestingWalletWithCliff is VestingWallet {
 
     error CurrentTimeIsBeforeCliff();
 
-    uint256 private immutable _cliffDuration;
-
-    modifier isAfterCliff() {
-        if (_isBeforeCliff()) {
-            revert CurrentTimeIsBeforeCliff();
-        }
-        _;
-    }
+    uint64 private immutable _cliffDuration;
 
     /**
      * @dev Set the cliff and owner.
      * @dev Set the beneficiary, start timestamp, and vesting duration within VestingWallet base class.
      */
-    constructor(uint64 cliffDurationSeconds) payable 
+    constructor(uint64 cliffDurationSeconds)
     {
```

Comment:
Guess I missed this the first time around. Have you run the linter lately?
```suggestion
    constructor(uint64 cliffDurationSeconds) {
```

Reply:
Linter didn't catch it -- maybe I need to configure it better.

Reply:
done